apps/site/src/_pages/legal/terms/dpa/00_index.mdx

```diff
@@ -169,7 +169,7 @@ Any additional or alternate instructions must be agreed to according to
+To the extent the Block Protocol uses or otherwise processes Personal Data subject to the GDPR for the Block Protocol’s legitimate business operations incident to delivery of the Online Services to Customer, the Block Protocol will comply with the obligations of an independent data controller under GDPR for such use. The Block Protocol is accepting the added responsibilities of a data “controller” under the GDPR for processing in connection with its legitimate business operations to: (a) act consistent with regulatory requirements, to the extent required under the GDPR; and (b) provide increased transparency to Customers and confirm the Block Protocol’s accountability for such processing. The Block Protocol employs safeguards to protect Personal Data in processing, including those identified in this DPA and those contemplated in Article 6(4) of the GDPR. With respect to processing of Personal Data under this paragraph, the Block Protocol makes the commitments set forth in the Standard Contractual Clauses set forth in [Attachment 1](/legal/terms/dpa/attachment-1.pdf) or [Attachment 2](/legal/terms/dpa/attachment-2.pdf) (as applicable); for those purposes, (i) any Block Protocol disclosure of Personal Data, as described in Annex IV to [Attachment 1](/legal/terms/dpa/attachment-1.pdf), that has been transferred in connection with the Block Protocol’s legitimate business operations is deemed a “Relevant Disclosure” and (ii) the commitments in Annex IV to [Attachment 1](/legal/terms/dpa/attachment-1.pdf) apply to such Personal Data.
```

Comment:
```suggestion
To the extent the Block Protocol uses or otherwise processes Personal Data subject to the GDPR for the Block Protocol’s legitimate business operations incident to delivery of the Online Services to Customer, the Block Protocol will comply with the obligations of an independent data controller under GDPR for such use. The Block Protocol is accepting the added responsibilities of a data “controller” under the GDPR for processing in connection with its legitimate business operations to: (a) act consistent with regulatory requirements, to the extent required under the GDPR; and (b) provide increased transparency to Customers and confirm the Block Protocol’s accountability for such processing. The Block Protocol employs safeguards to protect Personal Data in processing, including those identified in this DPA and those contemplated in Article 6(4) of the GDPR. With respect to processing of Personal Data under this paragraph, the Block Protocol makes the commitments set forth in the Standard Contractual Clauses set forth in [Attachment 1](#attachment-1) or [Attachment 2](#attachment-2) (as applicable); for those purposes, (i) any Block Protocol disclosure of Personal Data, as described in Annex IV to [Attachment 1](#attachment-1), that has been transferred in connection with the Block Protocol’s legitimate business operations is deemed a “Relevant Disclosure” and (ii) the commitments in Annex IV to [Attachment 1](#attachment-1) apply to such Personal Data.
```